modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/openapi/contributor/OpenAPIContributor.java

```diff
@@ -16,13 +16,12 @@
 
 import io.swagger.v3.oas.models.OpenAPI;
 
-import javax.ws.rs.core.UriInfo;
-
 /**
  * @author Luis Miguel Barcos
  */
 public interface OpenAPIContributor {
 
-	public void contribute(OpenAPI openAPI, UriInfo uriInfo) throws Exception;
+	public void contribute(String basePath, OpenAPI openAPI, String path)
```

Comment:
basePath and path are a bit confusing... what do they mean? do you think they can be wrapped in some kind of openAPIContext object?

Reply:
Yes! I agree, I have created it and it is much more beautiful :heart: